src/components/screens/Course.js

```diff
@@ -310,17 +304,11 @@ class Course extends React.PureComponent {
 	}
 
 	getDateFromTimeString = (timeString, currentDate) => {
-		if (currentDate === undefined) {
-			currentDate = new Date();
-		}
```

Comment:
Removing this may cause some other issues, since this function is used by other parts of the code.

Reply:
This function is only used in two places to assign time for startTime and endTime.

Reply:
Oh, my bad thought it was